### DISCUSSION TITLE
Allow bulk editing of rack from device

### DISCUSSION THREAD

**Discussion by themmini** [netbox-community/netbox#9441] [Ideas]
When bulk editing devices is there any reason why there's no ability to change what rack those devices are in?
Or "unrack". 

Use case: I am currently slowly offlining some hardware of a couple of racks. I can filter down the devices to about 40 devices in a particular site that are currently offline. I would like to be able to select all in the devices filter and unrack them from their current rack and assign to a new rack.

See example layout image.
![Netbox_editdevicesfields](https://user-images.githubusercontent.com/11536761/170900200-17852d2f-4e24-4162-86e7-0514af36a009.png)

**Comment by candlerb** [comment]
I note that if you were able to bulk-change the rack assignment of a device, it would also have to zero out the rack U position and front/rear face. Otherwise you would risk clashing with another device in the same rack at the same position.

**Reply by themmini** [reply]
I was thinking it could be used for non-racked or have a warning that it will “unrack” when moving racks if there is a conflict.

**Comment by jeremystretch** [comment]
This has been proposed in the past; see #4268.
